.github/workflows/CI-CD.yml

```diff
@@ -0,0 +1,47 @@
+name: IdlImporter CI/CD
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+      with:
+        fetch-depth: 0
+
+    - name: Install .NET Core
+      uses: actions/setup-dotnet@77a48bbb85cd3809790223e895ba4b6cbc110fe8
+      with:
+        dotnet-version: 6.0.x
+
+    - name: Restore
+      run: dotnet restore
+
+    - name: Build
+      run: dotnet build
+
+    - name: Pack
+      run: dotnet pack IDLImporter --include-symbols --no-restore --no-build -o .
+
+    - name: Pack
```

Comment:
We have `Pack` twice. Maybe name them `Pack IdlImporter` and `Pack IdlImpTool` ?